index.md

```diff
@@ -204,12 +205,17 @@ This is an overview of all Android versions and their corresponding identifiers
   <tr>
     <td>Level 17 <span class="subversion">Android 4.2</span></td>
     <td><code>JELLY_BEAN_MR1</code></td>
-    <td rowspan="2">2012</td>
+    <td rowspan="3">2012</td>
```

Comment:
"No data" cell needs a rowspan+1 too.

Reply:
Oh my bad, I've just now set the rowspan for the one instance of "No data" to 23. Does it look right now?

Reply:
👍

btw I'm only looking at [base](https://github.com/ebelinski/apilevels/blob/main/index.md) and [PR](https://github.com/Smu1zel/apilevels/blob/main/index.md)

Reply:
Oh that's good, should be good to merge then. Unless I left some fatal mistake that breaks the site, but that seems unlikely since I've seen other PRs and they add bullets like this in a similar fashion.

Reply:
Yeah, we just need the maintainer to look :)